serverless.yml

```diff
@@ -71,7 +71,7 @@ functions:
     timeout: 20  # Give aurora cluster time to cold start
 
   distributeDialogEvents:
-    handler: stopcovid/event_distributor.distribute_dialog_events
+    handler: stopcovid/event_distributor/event_distributor.distribute_dialog_events
```

Comment:
nit for the future. Possibly we put all the lambdas in a `lambda_handlers` directory outside of the stopcovid source.